tests/src/Kernel/Paragraphs/FactsFiguresTest.php

```diff
@@ -0,0 +1,216 @@
+
+  /**
+   * Tests the rendering of the paragraph type.
+   */
+  public function testRendering(): void {
+    // Create Fact paragraphs.
+    $paragraph_fact =
+    [
+      0 => Paragraph::create([
+        'type' => 'oe_fact',
+        'field_oe_icon' => 'box-arrow-up',
+        'field_oe_title' => '1529 JIRA Ticket',
+        'field_oe_subtitle' => 'Jira Tickets',
+        'field_oe_plain_text_long' => 'Nunc condimentum sapien ut nibh finibus suscipit vitae at justo. Morbi quis odio faucibus, commodo tortor id, elementum libero.',
+      ]),
+      1 => Paragraph::create([
+        'type' => 'oe_fact',
+        'field_oe_icon' => 'box-arrow-up',
+        'field_oe_title' => '337 Features',
+        'field_oe_subtitle' => 'Feature tickets',
+        'field_oe_plain_text_long' => 'Turpis varius congue venenatis, erat dui feugiat felis.',
+      ]),
+      2 => Paragraph::create([
+        'type' => 'oe_fact',
+        'field_oe_icon' => 'box-arrow-up',
+        'field_oe_title' => '107 Tests',
+        'field_oe_subtitle' => 'Test tickets',
+        'field_oe_plain_text_long' => 'Cras vestibulum efficitur mi, quis porta tellus rutrum ut. Quisque at pulvinar sem.',
+      ]),
+      3 => Paragraph::create([
+        'type' => 'oe_fact',
+        'field_oe_icon' => 'box-arrow-up',
+        'field_oe_title' => '5670 Variants',
+        'field_oe_subtitle' => 'Test variants',
+        'field_oe_plain_text_long' => 'Aliquam lacinia diam eu sem malesuada, in interdum ante bibendum.',
+      ]),
+      4 => Paragraph::create([
+        'type' => 'oe_fact',
+        'field_oe_icon' => 'box-arrow-up',
+        'field_oe_title' => '345 Dev Ticket',
+        'field_oe_subtitle' => 'Jira ticket',
+        'field_oe_plain_text_long' => 'Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Duis nec lectus tortor.',
+      ]),
+      5 => Paragraph::create([
+        'type' => 'oe_fact',
+        'field_oe_icon' => 'box-arrow-up',
+        'field_oe_title' => '43 Components',
+        'field_oe_subtitle' => 'Figma components',
+        'field_oe_plain_text_long' => 'Sed efficitur bibendum rutrum. Nunc feugiat congue augue ac consectetur.',
+      ]),
+    ];
+    // Create Facts and figures paragraph.
+    $paragraph = Paragraph::create([
+      'type' => 'oe_facts_figures',
+      'field_oe_title' => 'Fact and figures block',
+      'field_oe_link' => [
+        'uri' => 'https://www.readmore.com',
+        'title' => 'Read more',
+      ],
+      'oe_bt_layout' => 3,
+      'field_oe_paragraphs' => $paragraph_fact,
+    ]);
+    $paragraph->save();
+
+    // Testing: Facts and figures - Default layout.
+    $html = $this->renderParagraph($paragraph);
+    $crawler = new Crawler($html);
+
+    ini_set('xdebug.var_display_max_depth', '10');
+    ini_set('xdebug.var_display_max_children', '256');
+    ini_set('xdebug.var_display_max_data', '100024');
+    var_dump($crawler->filter('body')->html());
```

Comment:
Is the ini_set() and the var_dump() leftover code, or is there a reason for it?
If it is needed for some reason, I would expect a comment with an explanation.